petpals/models.py

```diff
@@ -147,3 +147,19 @@ def recent_photo_paths(self):
 
     def __repr__(self):
         return f"Profile('{self.name}', '{self.species}', '{self.subspecies}', '{self.color}', '{self.tagline}', '{self.biography}')"
+
+
+class Messages(db.Model):
+    message_id = db.Column(db.Integer, primary_key=True)
+    conversation_id = db.Column(db.Integer, nullable=False)
+    sender_username = db.Column(
+        db.String(18), db.ForeignKey("user.username"), nullable=False
+    )
+    recipient_username = db.Column(
+        db.String(18), db.ForeignKey("user.username"), nullable=False
+    )
+    time_sent = db.Column(db.TIMESTAMP, nullable=False)
+    message = db.Column(db.Text, nullable=False)
+
```

Comment:
A relationship should be added since you need to add `cascade='all, delete'` to the User relationships.
However, it needs to implemented [like this](https://docs.sqlalchemy.org/en/14/orm/join_conditions.html#handling-multiple-join-paths) since you have 2 foreign keys from the same table.